_fossfund/16-openstreetmap.md

```diff
@@ -0,0 +1,10 @@
+---
+title: OpenStreetMap
+logo: openstreetmap.svg
```

Comment:
This [image ](https://avatars.githubusercontent.com/u/16314049?s=200&v=4)should be added to 

https://github.com/microsoft/opensource.microsoft.com/tree/1dbc9603ccbda0f41c0535267ca502abe80aa0aa/assets/images/fossfund